test/README.md

```diff
@@ -0,0 +1,58 @@
+# Run integration tests from your local machine
+
+> This workflow  runs a single integration test from the local machine. It is a convenient alternate to GitHub actions; however, the limitation is that only one test can be run at a time.        
+
+## Setup
```

Comment:
* More than 3 step would be consider a pain and are there any way of automating it for scaling problems? This would increase in the pain in scaling knowledge across the team since they have to know both the main repository and the test repo. 
* Another thing to consider would be we would need to update manually every time there is a change in the main repository and the test one.